lib/_tls_wrap.js

```diff
@@ -300,12 +300,16 @@ var proxiedMethods = [
 ];
 
 // Proxy HandleWrap, PipeWrap and TCPWrap methods
-proxiedMethods.forEach(function(name) {
-  tls_wrap.TLSWrap.prototype[name] = function methodProxy(...args) {
+function makeMethodProxy(name) {
+  return function methodProxy(...args) {
```

Comment:
I'm curious why `...args` is being used when we're just using it with `.apply()` which should handle `arguments` just fine.